ambry-cloud/src/main/java/com/github/ambry/cloud/CloudStorageManager.java

```diff
@@ -108,6 +112,21 @@ public boolean controlCompactionForBlobStore(PartitionId id, boolean enabled) {
     throw new UnsupportedOperationException("Method not supported");
   }
 
+  @Override
+  public ClusterParticipant getPrimaryClusterParticipant() {
+    throw new UnsupportedOperationException("Method not supported");
+  }
+
+  @Override
+  public boolean isFileExists(PartitionId partitionId, String fileName) {
+    throw new UnsupportedOperationException("Method not supported");
+  }
+
+  @Override
+  public boolean isFilesExistForPattern(PartitionId partitionId, Pattern allLogSegmentFilesPattern) {
```

Comment:
Same as above.

Reply:
Same rationale as above. 
The interface cannot have a default impl as we are delegating the impl to DiskManager.